src/cart/crt_init.c

```diff
@@ -560,6 +560,19 @@ prov_settings_apply(bool primary, crt_provider_t prov, crt_init_options_t *opt)
 	g_prov_settings_applied[prov] = true;
 }
 
+int
+crt_protocol_info_get(const char *info_string, struct crt_protocol_info **protocol_info_p)
+{
+	D_ASSERT(sizeof(struct crt_protocol_info) == sizeof(struct na_protocol_info));
```

Comment:
_Static_assert is more appropriate here:)

Reply:
ah ok sure I can use that